pandas/core/indexes/base.py

```diff
@@ -141,12 +141,10 @@ class Index(IndexOpsMixin, PandasObject):
     _join_precedence = 1
 
     # Cython methods
-    _arrmap = libalgos.arrmap_object
```

Comment:
so I think you can also remove some of the cython code which implements (not float/object though)

Reply:
arrmap_object is used in core.ops

Reply:
right, but can remve the rest of the code. pls add to the list.

Reply:
> right, but can remve the rest of the code. pls add to the list.

The rest of what code?

Reply:
arrmap in cython